homeassistant/components/tuya/sensor.py

```diff
@@ -818,6 +825,35 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             native_unit_of_measurement=UnitOfElectricPotential.VOLT,
             subkey="voltage",
         ),
+        TuyaSensorEntityDescription(
+            key=DPCode.ADD_ELE,
```

Comment:
These don't update?

As far as I am aware, the `add_ele` doesn't update. And if it updates, it needs to add the current values (which we cannot do). Additionally, if pushed, it is every 1kWh, which is kinda useless.

Reply:
1. The "TOTAL_FORWARD_ENERGY" and "FORWARD_ENERGY_TOTAL" is exclusive, there is no device can have both of these dpcode, so there will not be duplicate sensor case.
2. The "add_ele" will receive updated via MQTT, so it's not still. Our customers want to present this dpcode, if it's not supported, do you have any suggestions if we want to present this value?

Reply:
> The "add_ele" will receive updated via MQTT, so it's not still. Our customers want to present this dpcode, if it's not supported, do you have any suggestions if we want to present this value?

There is now way to handle this value, hence it was never added. This subject had been brought up a couple of times, and all related PRs have been closed.

Did you test this sensor? How did it work? Can you describe what will happen and how it will handle the resetting behavior this one brings?

../Frenck

Reply:
It looks like this:
![example](https://github.com/home-assistant/core/assets/907831/43d7d3f7-1a0a-4ffa-8a70-100699b9391a)

Reply:
I understood that from the code... that is not what I was asking.

../Frenck